bumble/hci.py

```diff
@@ -6876,12 +6795,15 @@ class HCI_LE_CS_Test_End_Complete_Event(HCI_LE_Meta_Event):
 
 
 # -----------------------------------------------------------------------------
-@HCI_Event.event([('status', STATUS_SPEC)])
+@HCI_Event.dataclass_event
+@dataclasses.dataclass
 class HCI_Inquiry_Complete_Event(HCI_Event):
     '''
     See Bluetooth spec @ 7.7.1 Inquiry Complete Event
     '''
 
+    status: int = field(metadata=metadata(STATUS_SPEC))
```

Comment:
we could define `STATUS_METADATA=metadata(STATUS_SPEC)`

Reply:
I think this could be merged with `SpecableEnum` later? Since it's not too long here.

Finally we may replace all `*_SPEC` with `SpecableEnum`.